tests/TestFixtures/TNSTestNativeCallbacks.h

```diff
@@ -12,6 +12,10 @@
 
 @interface TNSTestNativeCallbacks : NSObject
 
+typedef void (^TNSBlock)();
+
+@property(assign, nonatomic) TNSBlock block;
```

Comment:
I would suggest to move this to `TNSObjCTypes.m` where we have tests like:
* `methodWithNSDate`
* `methodWithNSArray`
* `methodWithNSDictionary`
* `methodWithNSData`

which test the same thing (that the object is the same).

Also removing the property from the header won't make the test fail, an is a bit brittle.